.github/workflows/build.yml

```diff
@@ -30,185 +34,295 @@ jobs:
         echo "CABAL_BUILDDIR=$CABAL_BUILDDIR"
         echo "CABAL_BUILDDIR=$CABAL_BUILDDIR" >> $GITHUB_ENV
 
+    - name: Set cache version
+      run: echo "CACHE_VERSION=9w76Z3Q" >> $GITHUB_ENV
+
     - name: Install pkgconfiglite
       if: matrix.os == 'windows-latest'
       run: choco install -y pkgconfiglite
 
-    - name: Install libsodium
+    - name: Install libsodium (Windows)
       if: matrix.os == 'windows-latest'
+      env:
+        RUNNER_TEMP: ${{ runner.temp }}
       run: |
+        echo "RUNNER_TEMP=$RUNNER_TEMP"
+        cd "$RUNNER_TEMP"
+
+        RUNNER_TEMP_FWD="$(echo "$RUNNER_TEMP" | sed 's|\\|/|g')"
+
         curl -Ls \
           --connect-timeout 5 \
           --max-time 10 \
           --retry 5 \
           --retry-delay 0 \
           --retry-max-time 40 \
           https://download.libsodium.org/libsodium/releases/libsodium-1.0.18-mingw.tar.gz -o libsodium-1.0.18-mingw.tar.gz
-        tar zxvf libsodium-1.0.18-mingw.tar.gz
 
-        sed -i "s|/d/a/1/s/|D:/a/cardano-node/cardano-node/|g" libsodium-win64/lib/pkgconfig/libsodium.pc
+        tar zxvf libsodium-1.0.18-mingw.tar.gz
+        sed -i "s|/d/a/1/s/|$RUNNER_TEMP_FWD\/|g" libsodium-win64/lib/pkgconfig/libsodium.pc
 
         export PKG_CONFIG_PATH="$(readlink -f libsodium-win64/lib/pkgconfig)"
+        echo "PKG_CONFIG_PATH=$PKG_CONFIG_PATH"
         echo "PKG_CONFIG_PATH=$PKG_CONFIG_PATH" >> $GITHUB_ENV
 
         export LIBSODIUM_PATH="$(readlink -f libsodium-win64/bin | sed 's|^/d|D:|g' | tr / '\\')"
         echo "LIBSODIUM_PATH=$LIBSODIUM_PATH"
         echo "$LIBSODIUM_PATH" >> $GITHUB_PATH
 
-    - name: Install libsodium
+    - name: Install libsodium (MacOS)
       if: matrix.os == 'macos-latest'
       run: brew install libsodium
 
-    - name: Configure to use libsodium
+    - name: Install build environment
```

Comment:
I'd call it `Install libsodium (Linux)` to be consistent with other similar steps.

Reply:
Done.  Thanks!